CHANGELOG.md

```diff
@@ -1,3 +1,87 @@
+## 1.1.0-rc1 (September 31, 2022)
```

Comment:
Would we want to put this in reverse chronological order? Also, did we officially decide to deprecate inline changelog notices? If not, I would expect to see 1.2.0 here.

Reply:
I don't know what the rule is, the script always adds to the top of the file